### ISSUE TITLE
Disable attachment features if there is no attachment server 

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#2585]
> As a test, we hit our DEV instance of 7.7.5 at: http://herb.specify-dev.lsa.umich.edu/specify/attachments/ to see if attachments load.
We found that Specify's attachment view attempts to render and request images.
From the screenshot below the images do not load [as intended].

![image](https://user-images.githubusercontent.com/37256050/207101214-61756ddd-98e4-4a89-b20d-93cebfca6103.png)

> Other attachment views also render attachment-related controls but operations fail or are not possible due to the attachment server being disabled.

![image](https://user-images.githubusercontent.com/37256050/207101227-a551e3df-f14e-4ea3-942d-5c41f1a074dc.png)

> If you click save though it does not succeed in creating the empty attachments [this is good].
>
>I suppose this ticket is just to discuss whether all of these views should be disabled if the attachment server is not configured.

Matthew makes a good point– areas where attachments could be attached/viewed should either be disabled or be better at reporting there is no attachment server rather than attempting to upload or display attachments that cannot be reached. While this is uncommon, I imagine this would be helpful for users to communicate the disconnect.

**Reported By:** Matthew Cruz at UMich (Email)

**Comment by maxxxxxdlp** [comment]
The attachments view is removed from the top level menu if attachments server is not configured. Probably should also add protection against users directly going to that URL

**Comment by mcruz-umich** [comment]
Yes, I should have been more explicit in my description and clearly stated that I noticed the "Attachments" view button was removed from the interface since we do not have the attachment server registered, however as a test I tried hitting the url directly to see what would happen.

In the second example however, I was viewing a CollectionObject record and I clicked on the "CO Attachment Paperclip" button that shows 1 attachment is available. My suggestion in that case would be to not render the paperclip button since the attachment server is not available.
